plugin.js

```diff
@@ -47,17 +47,17 @@ function onReqHandlerWrapper (options) {
 }
 
 function plugin (fastify, options, next) {
-  const secret = options ? options.secret || '' : ''
+  const pluginSecret = options ? options.secret || '' : ''
 
   fastify.decorateRequest('cookies', {})
   fastify.decorateReply('setCookie', function setCookieWrapper (name, value, options) {
-    return fastifyCookieSetCookie(this, name, value, options, secret)
+    return fastifyCookieSetCookie(this, name, value, options, pluginSecret)
```

Comment:
I would add the same logic `secret || pluginSecret` to keep the same feature as per unsign